klaviyo/campaigns.py

```diff
@@ -0,0 +1,48 @@
+from .api_helper import KlaviyoAPI
+
+
+class Campaigns(KlaviyoAPI):
+    CAMPAIGNS = 'campaigns'
+    CAMPAIGN = 'campaign'
+    RECIPIENTS = 'recipients'
+
+    def get_campaigns(self, page=None, count=None):
```

Comment:
we may want to specify a default page and default count like we do in other api's as well, also the docstring says these are strings but i think page and count should be int? https://apidocs.klaviyo.com/reference/campaigns

Reply:
@remstone7